app/views/admin/manage_users/index.html.erb

```diff
@@ -17,6 +20,9 @@
         <td class="govuk-table__cell govuk-!-width-one-third" scope="row">
           <%= user.created_at.strftime('%e %b %Y') %>
         </td>
+        <td class="govuk-table__cell govuk-!-width-one-third" scope="row">
+          <%= link_to 'Remove user', admin_manage_users_path(id: user.id, user_to_remove: true), class: "govuk-link" %>
```

Comment:
I feel this text should be red to match the other places where we have a destructive remove button/link

Reply:
I don't think it needs to be red. It's not immediate destruction.
And we haven't kept to a pattern:
IPs' remove link isn't red but team members' is.
Unless this is going to change?

Reply:
There does appear to be some inconsistency across blue vs red text for destructive actions.

Given there's no guidance around link colours on types of actions beyond "links are blue", and "buttons are green", let's stick with blue for now (unless we already have some designer input around this)